service-commander.spec

```diff
@@ -99,7 +100,7 @@ fi
 - feature: Several new options for 'sc_install_defaults'
 - feature: New '--ignore-groups' option for 'sc'
 - feature: Allow service path names to be relative
-- feature: preinstall common system services
+- feature: preinstall crond, MariaDB, and Zend DBi
```

Comment:
How about PostgreSQL - is it too difficult to include in the OSS common services? Or maybe we can do that when we have resolved the PR "dispute"... :grin: 

By the way, there is one comma too many...

Reply:
Let's continue that discussion in #142